issue_tracker/settings.py

```diff
@@ -0,0 +1,138 @@
+"""
+Django settings for issue_tracker project.
+
+Generated by 'django-admin startproject' using Django 4.2.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.2/ref/settings/
+"""
+
+from pathlib import Path
+import os
+from dotenv import load_dotenv
+
+# loads environment variable from .env file
+load_dotenv()
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-ue%w+0(*zuxzaq_nxu1p%*)(l%j=&f!wum#ak-r@n!*#fidajd'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = []
+
+# Application definition
+
+INSTALLED_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+]
+
+MIDDLEWARE = [
+    'django.middleware.security.SecurityMiddleware',
+    'django.contrib.sessions.middleware.SessionMiddleware',
+    'django.middleware.common.CommonMiddleware',
+    'django.middleware.csrf.CsrfViewMiddleware',
+    'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'django.contrib.messages.middleware.MessageMiddleware',
+    'django.middleware.clickjacking.XFrameOptionsMiddleware',
+
+    # EXCEPTION HANDLER MIDDLEWARE
+    'issue_tracker.middleware.ExceptionHandleMiddleware'
```

Comment:
We are not handling exception here. We are notifying when issue will arise. So please make sure on proper naming

Reply:
Apologies for the confusion. If the objective is to notify when issues arise without explicitly handling the exceptions, an appropriate middleware name could be "**IssueNotificationMiddleware**" or "**ErrorNotificationMiddleware**."

Reply:
Changed the middleware name as per the suggestion.